tests/test_utils.py

```
@@ -2,325 +2,304 @@
+    @patch("subprocess.run")
     def test_run_command_with_timeout(self, mock_run):
         """Test command execution with timeout"""
-        mock_run.return_value = Mock(
-            returncode=0,
-            stdout='output',
-            stderr=''
-        )
-        
+        mock_run.return_value = Mock(returncode=0, stdout="output", stderr="")
+
         rc, stdout, stderr = run_command('echo "test"', timeout=30)
```

Ruff (0.13.1)

Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)